modules/hal_adi/Kconfig

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Analog Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Place holder if/when Kconfig is needed
```

Comment:
> Also this file requires to github action works

Which action was failing without this file?

Reply:
It was [doc-build.yml](https://github.com/zephyrproject-rtos/zephyr/actions/runs/8940573961/job/24559300517#step:9:101).
But I believe you are asking me to update commit to mention it.

Thanks commit message updated.

Reply:
I think instead of introducing the placeholder file you could set `files:` to be empty in `MAINTAINERS.yml`. e.g.,
```yml
"West project: hal_adi":
  status: maintained
  maintainers:
    - MaureenHelm
  collaborators:
    - ozersa
  files: []
  labels:
    - "platform: ADI"
 ```

Reply:
Thanks, both of them are fine.
Updates as your suggestion.